.github/workflows/cypress-parallel-2.yml

```diff
@@ -0,0 +1,20 @@
+name: cypress-2
```

Comment:
I have a feeling we can do this with the `matrix` stuff in github actions rather than breaking out into different workflows? https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategy

Reply:
Amazing. Thanks for pointing out that feature, I've not seen it before and it is exactly what I needed!